src/Items/ItemFishingRod.h

```diff
@@ -236,9 +239,11 @@ class cItemFishingRodHandler :
 				{
 					return true;
 				}
-				Vector3d FloaterPos = FloaterInfo.GetPos();
+				Vector3d FloaterPos = FloaterInfo.GetBitePos();
+				const float FISH_SPEED_MULT_XZ = 2.7f;
+				const float FISH_SPEED_MULT_Y = 2.0f;
```

Comment:
I tweaked it again a little bit but yes, for the most part it scales well for all _reasonable_ heights and distances. Although, the minecraft wiki says that moving 32 blocks away should break the line, but that doesn't happen. That's probably the next thing I'll work on.